include/swift/AST/Attr.def

```diff
@@ -136,9 +135,7 @@ CONTEXTUAL_SIMPLE_DECL_ATTR(optional, Optional,
 SIMPLE_DECL_ATTR(dynamicCallable, DynamicCallable,
   OnNominalType,
   6)
-SIMPLE_DECL_ATTR(noreturn, NoReturn,
-  OnFunc | OnAccessor,
-  7)
+// NOTE: 6 is unused
```

Comment:
Looks like 7 is unused? Or maybe I’m misreading.

Reply:
You’re absolutely right.